packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -213,6 +223,20 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
         throw new Error("Cannot attach container. Container is not in detached state");
     }
 
+    /**
+     * {@inheritDoc IFluidContainer.connect}
+     */
+    public async connect(): Promise<void> {
+        return this.container.connect?.();
```

Comment:
The syntax here isn't great. The alternative would be to wait to make these changes once `connect()`/`disconnect()` are made mandatory on `IContainer`. However, I felt it made sense to make the functions mandatory for both `IContainer` and `IFluidContainer` in a separate future PR (since they are breaking changes), and then update the syntax here after.